ambry-clustermap/src/main/java/com/github/ambry/clustermap/DatacenterInitializer.java

```diff
@@ -102,15 +78,10 @@ public void start() {
     Utils.newThread(() -> {
       try {
         DcInfo dcInfo;
-        switch (dcZkInfo.getReplicaType()) {
-          case DISK_BACKED:
-            dcInfo = initializeHelixDatacenter();
-            break;
-          case CLOUD_BACKED:
-            dcInfo = initializeCloudDatacenter();
-            break;
-          default:
-            throw new UnsupportedOperationException("Unknown replica type: " + dcZkInfo.getReplicaType());
+        if (dcZkInfo.getReplicaType() == ReplicaType.DISK_BACKED) {
```

Comment:
This change is part of https://github.com/linkedin/ambry/pull/2236. Will rebase it once that PR is merged